src/components/pages/home/homepage-data.ts

```diff
@@ -1,5 +1,5 @@
-const homepageData = [
-  {
+const homepageData = {
```

Comment:
as an object we don't need an `id` prop since we can just access it as a prop, which is in fact an id!